trefoil/netcdf/tests/test_crs.py

```diff
@@ -34,7 +34,6 @@ def test_get_crs(tmpdir):
 
     out_data = CRS.from_string(out_proj4).to_dict()
 
-    assert len(out_data) == 8  # There should be 8 parameters

```

Comment:
This assertion failed (6 != 8), but it's hard to know what 8 things were wanted. More direct assertions of the contents of `out_data` follow.

Reply:
Weird, when I run this code in the interpreter, I get a dict with 10 items 🤔 Anyway, I think it's okay to remove this assertion, since in the next one, we're verifying the CRS from the dataset is what we expect.

Reply:
Sorry, 10 is right. I remembered a difference of 2 when writing the comment and went the wrong direction.